code/datums/diseases/advance/symptoms/skin.dm

```diff
@@ -181,3 +181,137 @@ BONUS
+	stealth = -3 //You are basically growing these weird Egg shits on your skin, this is not stealthy in the slightest
+	resistance = 1
+	stage_speed = 0
+	transmittable = 2 //The symptom is in it of itself meant to spread
+	level = 9
+	severity = -1
+	base_message_chance = 50
+	symptom_delay_min = 60
+	symptom_delay_max = 105
+	threshold_desc = "<b>Transmission 12:</b> Eggs and Egg Sacs contain all diseases on the host, instead of just the disease containing the symptom.<br>\
+					  <b>Transmission 16:</b> Egg Sacs will 'explode' into eggs after a period of time, covering a larger area with infectious matter.<br>\
+					  <b>Resistance 10:</b> Eggs and Egg Sacs contain more healing chems.<br>\
+					  <b>Stealth 6:</b> Eggs and Egg Sacs become nearly transparent, making them more difficult to see.<br>\
+					  <b>Stage Speed 10:</b> Egg Sacs fall off the host more frequently."
+
+/datum/symptom/skineggs/severityset(datum/disease/advance/A)
+	. = ..()
+	if(A.properties["resistance"] >= 10)
+		severity -= 1
+	if(A.properties["transmittability"] >= 12)
+		severity += 1
+	if(A.properties["transmittability"] >= 16)
+		severity += 1
+	if(A.properties["stealth"] >= 6)
+		severity += 1
+
+/datum/symptom/skineggs/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.properties["resistance"] >= 10)
+		thresholds["bigheal"] = TRUE
+	if(A.properties["transmittability"] >= 12)
+		thresholds["alldisease"] = TRUE
+	if(A.properties["transmittability"] >= 16)
+		thresholds["eggsplosion"] = TRUE //Haha get it?
+	if(A.properties["stealth"] >= 6)
+		thresholds["sneaky"] = TRUE
+	if(A.properties["stage_rate"] >= 10)
+		symptom_delay_min -= 10
+		symptom_delay_max -= 20
+
+
+/datum/symptom/skineggs/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/M = A.affected_mob
+	var/list/diseases = list(A)
+	switch(A.stage)
+		if(5)
+			if(thresholds["alldisease"])
+				for(var/datum/disease/D in M.diseases)
+					if((D.spread_flags & DISEASE_SPREAD_SPECIAL) || (D.spread_flags & DISEASE_SPREAD_NON_CONTAGIOUS) || (D.spread_flags & DISEASE_SPREAD_FALTERED))
+						continue
+					if(D == A)
+						continue
+					diseases += D
+			new /obj/item/reagent_containers/food/snacks/eggsac(diseases, thresholds["eggsplosion"], thresholds["sneaky"], thresholds["bigheal"])
+/obj/item/reagent_containers/food/snacks/eggsac
+	name = "Fleshy Egg Sac"
+	desc = "A small Egg Sac which appears to be made out of someone's flesh!"
+	customfoodfilling = 0 //Not Used For Filling
+	icon_state = "icons/obj/food/food/eggsac.dmi"
+	bitesize = 4
+	var/list/diseases = list()
+	var/sneakyegg
+	var/bigheal
+
+//Constructor
+/obj/item/reagent_containers/food/snacks/eggsac/New(var/list/disease, var/eggsplodes, var/sneaky, var/largeheal)
+	for(var/datum/disease/D in disease)
+		diseases += D
+	if(largeheal)
+		src.add_initial_reagents(list(/datum/reagent/medicine/bicaridine = 20, /datum/reagent/medicine/tricordrazine = 10))
```

Comment:
Do trekchems still have overdoses? May be wise to use something that doesnt if so

Reply:
They do, however, the chems in them are under OD. This is in addition to the fact that only a portion is added with each bite.

However, since these are basically infinite sources of portable healing, it's fair to have an OD to stop butt chugging 20 egg sacs to outheal space dragons.